controllers/serviceexport/syncstatustracker.go

```diff
@@ -0,0 +1,19 @@
+package serviceexport
+
+// TO-DO (chenyu1): Refactor this utility with generics when Go 1.18 becomes available
+
+type SvcSyncStatusTracker interface{}
```

Comment:
Forget to mention, let's remove the interface for now.

"Do not define interfaces before they are used: without a realistic example of usage, it is too difficult to see whether an interface is even necessary, let alone what methods it ought to contain."

https://github.com/golang/go/wiki/CodeReviewComments#interfaces

Reply:
Oh, actually this interface is in use (line 74 in `controller.go`) :)

Reply:
Interface moved as discussed.